### ISSUE TITLE
Clicking "less accessory slots" causes page to blank

### ISSUE THREAD

**Issue by thePalindrome** [gmiclotte/gear-optimizer#2]
Doing certain actions (like deleting slot 0 or reducing the number of accessory slots) causes the entire page to blank, with the following error spammed in the console:
```
TypeError: A is undefinedEquipTable.js:28:8
    gA EquipTable.js:28
    value EquipTable.js:154
    React 9
        ji
        Pi
        Mi
        Va
        Ka
        jl
        Cl
        Ll
        En
    En self-hosted:1035
```

It also spams this error afterwards

```
TypeError: "A is undefined"
    gA EquipTable.js:28
    value EquipTable.js:154
    React 9
        ji
        Pi
        Mi
        Va
        Ka
        jl
        Cl
        Ll
        En
react-dom.production.min.js:4408
    React 7
    unstable_runWithPriority scheduler.production.min.js:274
    React 5
```

This is firefox 67.0

**Comment by thePalindrome** [comment]
Update, after re-selecting "Optimize Equipment" and updating the acc slots, I can now properly reduce the number of slots.

**Comment by gmiclotte** [comment]
Deleting slot 0 should be resolved by 4731963 . The acc slot issue is a side effect of #3.

**Comment by gmiclotte** [comment]
If the bug still happens, then it should be because your local storage is still affected, just clear it. Alternatively you can save at least 2 different loadouts before deleting any. Doing this once should permanently resolve the issue.
